From 2d56556279ca7f7c230a1a273ceb0a7f7362b6e9 Mon Sep 17 00:00:00 2001
From: Shawn Landden <shawn@churchofgit.com>
Date: Tue, 11 Feb 2014 19:01:47 -0800
Subject: [PATCH] remove After=syslog.target, obsolete

W: connman-vpn: systemd-service-file-refers-to-obsolete-target
lib/systemd/system/connman-vpn.service syslog.target
N:
N:    The systemd service file refers to an obsolete target.
N:
N:    Some targets are obsolete by now, e.g. syslog.target or
dbus.target. For
N:    example, declaring After=syslog.target is unnecessary by now because
N:    syslog is socket-activated and will therefore be started when needed.
N:
N:    Severity: normal, Certainty: certain
N:
N:    Check: systemd, Type: binary
---
 src/connman.service.in     | 1 -
 vpn/connman-vpn.service.in | 1 -
 2 files changed, 2 deletions(-)

diff --git a/src/connman.service.in b/src/connman.service.in
index 2e9e4d5..e88bb88 100644
--- a/src/connman.service.in
+++ b/src/connman.service.in
@@ -1,6 +1,5 @@
 [Unit]
 Description=Connection service
-After=syslog.target
 
 [Service]
 Type=dbus
diff --git a/vpn/connman-vpn.service.in b/vpn/connman-vpn.service.in
index ec02a86..a827e5f 100644
--- a/vpn/connman-vpn.service.in
+++ b/vpn/connman-vpn.service.in
@@ -1,6 +1,5 @@
 [Unit]
 Description=ConnMan VPN service
-After=syslog.target
 
 [Service]
 Type=dbus
-- 
1.9.rc0.143.g6fd479e

